src/components/forgetPass/resetPassword.js

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import * as image from '../image/image.js';
+import { useSelector, useDispatch } from 'react-redux';
+import * as actionsReset from '../../actions/axiosResetPass.js';
+import { useLocation } from 'react-router-dom';
+import AlertForm from '../alert/alert.js';
+
+function ResetPassword(props) {
+  const dispatch = useDispatch();
+  const queryString = require('query-string');
+  const headersUrl = queryString.parse(useLocation().search);
+
+  const [displayAlert, setDisplayAlert] = useState('none');
+  const [password,setPassword] = useState('');
+  const [passConf ,setPassConf] = useState('');
+
+  const getPassReset = (value) =>{
+    setPassword(value);
+  }
+  const getPassResetConf = (value) =>{
+    setPassConf(value);
+  }
+  const clickToSend = (event) => {
+    event.preventDefault();
+    if(password === passConf && password !== ''&& passConf !== ''){
+      dispatch(actionsReset.sendResetPassAPI(password,  headersUrl))
+    }else{
+      setDisplayAlert('');
+    }
+    
+  }
+
+  return (
+    <div className='container' style={{ width: '40%' }}>
+      <AlertForm  displayAlert={displayAlert} setDisplayAlert={setDisplayAlert} message={'change pass is not successful!'}/>
```

Comment:
message ở đây là pass vs passConfirmation phải giống nhau.
toàn bộ message + placeholder (nói chung mọi text) em tách riêng thành 1 file, dùng package `i18next` để set ngôn ngữ -> cái này làm ở pull sau cũng okie (sau khi merge pull này)